docs/configuration.md

```diff
@@ -72,7 +72,7 @@ However it is strongly recommended to use `$name` in your output template, as `o
 
 You can turn off caching if you like. By default caching is on. FuseBox will create a folder `.fusebox` in your project path, and store related files. Don't forget to add it to .gitignore.
 
-note: If things go wrong or things are not updating, delete the `.fusebox` folder to force clear the cache.
```

Comment:
please revert all `note:`
It will look like [this](http://fuse-box.org/page/configuration#home-directory):
![screenshot 2017-05-15 09 32 37](https://cloud.githubusercontent.com/assets/1629088/26046949/8920453e-3951-11e7-9356-d79ad7ff6880.png)